src/main/java/bookstore/controller/BookController.java

```diff
@@ -29,7 +38,19 @@ public BookDto findById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+    public BookDto createBook(@RequestBody BookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @PutMapping("/{id}")
+    public BookDto updateBook(@PathVariable Long id, @RequestBody BookRequestDto requestDto) {
+        Book book = bookMapper.toModel(requestDto);
```

Comment:
Be consistent, you implement the mapping logic in service, so replace it there. 

About using mapper in controller or service layer, let's read some articles, for example [here](https://stackoverflow.com/questions/47457009/which-is-best-layer-to-place-mapper-code-service-layer-or-controller-layer)